### PR TITLE
Power optimization flag changes

### PR DESCRIPTION
**Pull request by kreedula**
common PR : https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/894

### CODE REVIEW

**Comment by shahzaibj** [comment]
@kreedula Builds are failing. I believe it is because https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1003 needs to merge in.

File: msal/src/main/java/com/microsoft/identity/client/PublicClientApplicationConfiguration.java [hunk 1]
```diff
@@ -421,6 +433,7 @@ void mergeConfiguration(PublicClientApplicationConfiguration config) {
         this.mLoggerConfiguration = config.mLoggerConfiguration == null ? this.mLoggerConfiguration : config.mLoggerConfiguration;
         this.webViewZoomControlsEnabled = config.webViewZoomControlsEnabled == null || config.webViewZoomControlsEnabled;
         this.webViewZoomEnabled = config.webViewZoomEnabled == null || config.webViewZoomEnabled;
+        this.powerOptCheckEnabled = config.powerOptCheckEnabled == null || config.powerOptCheckEnabled;
```
Comment (shahzaibj) [inline msal/src/main/java/com/microsoft/identity/client/PublicClientApplicationConfiguration.java:436]: So it seems like if they don't declare anything in config then the power opt check is enabled by default. Do we want that? Or should it be disabled by default?

Comment (kreedula) [reply msal/src/main/java/com/microsoft/identity/client/PublicClientApplicationConfiguration.java:436]: yes, given we removed `UsageStats.isAppInactive` check , I think we would be safer not to enable this by default, we can also some data from Teams to make a decision.

Comment (shahzaibj) [reply msal/src/main/java/com/microsoft/identity/client/PublicClientApplicationConfiguration.java:436]: @kreedula So you are going to update the code? It seems enabled by default at the moment

Comment (kreedula) [reply msal/src/main/java/com/microsoft/identity/client/PublicClientApplicationConfiguration.java:436]: @shahzaibj Apologies, edited the comment above

**Pull request merged**